apps/stats/src/utils/chart-helpers.ts

```
@@ -37,3 +32,24 @@ export const calculateYAxisWidth = (ticks: number[], formatter: (value: number)
+    const today = new Date();
+    const startDate = new Date(today);
+    startDate.setDate(today.getDate() - range);
+    return {today, startDate};
+};
+
+/**
+ * Converts a country code to corresponding flag emoji
+ */
+export function getCountryFlag(countryCode:string) {
+    if (!countryCode || countryCode === null || countryCode.toUpperCase() === 'ᴺᵁᴸᴸ') {
+        return '🏳️';
```

Biome (1.9.4)

[error] This expression always returns false, because the string is converted and will never match

This call convert the string to upper case

... but this value is not in upper case

Unsafe fix: Use upper case string value.


(lint/correctness/noStringCaseMismatch)